presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/TableScanMatcher.java

```diff
@@ -34,17 +34,27 @@
 {
     private final String expectedTableName;
     private final Optional<Map<String, Domain>> expectedConstraint;
+    private final Optional<Boolean> hasTableLayout;
```

Comment:
Maybe `expectedTableLayout`, And instead of `Boolean` there could be some layout matcher?